### ISSUE TITLE
Call fails with mixed level regions

### ISSUE THREAD

**Issue by mountainMath** [mountainMath/cancensus#168]
The following call fails.

```
get_census("CA16",regions=list(PR="59",CMA="59933"),vectors="v_CA16_1")
```

Additionally, the following call produces a warning message.

```
get_census("CA16",regions=c(C="01"),vectors="v_CA16_1")
```

**Comment by dshkol** [comment]
Was it ever working before?

**Comment by mountainMath** [comment]
I believe so. Might have been a victim of the refactoring I did in the last release. Either way, needs fixing. I will take a look tomorrow.

Also a good reminder that we really need tests.

**Comment by mountainMath** [comment]
This is fixed in the v0.4.3 branch. https://github.com/mountainMath/cancensus/pull/169
